apps/prairielearn/src/pages/elementFiles/elementFiles.js

```diff
@@ -24,67 +23,81 @@ const sql = sqldb.loadSqlEquiv(import.meta.url);
 const EXTENSION_WHITELIST = ['.js', '.css'];
 const CLIENT_FOLDER = 'clientFilesElement';
 
-router.get(
-  '/*',
-  asyncHandler(async (req, res) => {
-    const filename = req.params[0];
-    const pathSpl = path.normalize(filename).split('/');
-    const valid =
-      pathSpl[1] === CLIENT_FOLDER ||
-      EXTENSION_WHITELIST.some((extension) => filename.endsWith(extension));
-    if (!valid) {
-      throw new HttpStatusError(404, 'Unable to serve that file');
-    }
+export default function (options = { publicQuestionEndpoint: false, static: false }) {
```

Comment:
JSDoc comment? I'm specifically wondering what `static` means here.

Reply:
`static` was supposed to mean that it is a built-in PraireLearn element, not a course custom element. Previously, this was done just by checking if `res.locals.course` exists, but that doesn't work anymore, because for `/pl/public` URLs, `res.locals.course` won't exist, even if it is supposed to be using a course element!

Maybe `static` is a bad name for it. Should I call it `coreElement` instead? something else?

Reply:
renamed to `coreElements`